client/.eslintrc.json

```diff
@@ -1,10 +1,11 @@
 {
   "env": {
     "es6": true,
-    "browser": true
+    "browser": true,
+    "jest/globals": true
   },
   "extends": ["eslint:recommended", "plugin:react/recommended", "prettier"],
-  "plugins": ["react"],
+  "plugins": ["react", "jest"],
```

Comment:
We will also have to extend `plugin:jest/recommended` to enforce some rules.
Or they have to be specified manually.
If you do extend, keep prettier at last.

Reply:
Done.